GoogleSignIn/Tests/Unit/GIDGoogleUserTest.m

```diff
@@ -149,6 +149,20 @@ - (void)testUpdateAuthState_tokensAreNotChanged {
   XCTAssertIdentical(user.refreshToken, refreshTokenBeforeUpdate);
 }
 
+- (void)testFetcherAuthorizer {
```

Comment:
Let's add a test to check that multiple property accesses return the same instance.

Reply:
Added the test.